Sky Atlas/Sky Atlas.html

```diff
@@ -25212,20 +25169,20 @@ <h1>Agent Scope Database</h1>
           <td>The documents herein specify requirements related to Spark's Total Risk Capital (TRC) management.</td>
         </tr>
         <tr>
-          <td>
-            <dfn>Phoenix Labs' Operation Of Spark Liquidity Layer And Agreement Regarding Risk Tolerance Ratio</dfn>
+            <dfn>Spark's Operation Of Spark Liquidity Layer And Agreement Regarding Risk Tolerance Ratio</dfn>
+            <dfn>Spark’s Operation Of Spark Liquidity Layer And Agreement Regarding Risk Tolerance Ratio</dfn>
           </td>
           <td>Spark</td>
           <td>Core</td>
-          <td>Phoenix Labs will continue to operate the Spark Liquidity Layer and agrees to stay at or below a 90% Risk Tolerance Ratio. See <dfn>A.3.3 - Risk Tolerance Ratio</dfn>.</td>
+          <td>Spark will continue to operate the Spark Liquidity Layer and agrees to stay at or below a 90% Risk Tolerance Ratio. See <dfn>A.3.3 - Risk Tolerance Ratio</dfn>.</td>
         </tr>
         <tr>
-          <td>
-            <dfn>Phoenix Labs' Total Risk Capital (TRC) Management Processes</dfn>
+            <dfn>Spark's Total Risk Capital (TRC) Management Processes</dfn>
```

Comment:
The \<td> tag has been deleted incorrectly

```suggestion
          <td>
            <dfn>Spark's Total Risk Capital (TRC) Management Processes</dfn>
```